### PR TITLE
adding alb

### PR DESCRIPTION
**Pull request by ramiray**

### CODE REVIEW

File: terraform/modules/networking/nacl-private.tf [hunk 1]
```diff
@@ -29,6 +29,30 @@ resource "aws_network_acl_rule" "private_ingress_ephemeral_vpc" {
   to_port        = 65535
 }
 
+resource "aws_network_acl_rule" "private_ingress_https_vpc" {
```
Comment (raghunadhpokkalath) [inline terraform/modules/networking/nacl-private.tf:32]: Do we need add rule for 80 here since we have enabled 80 port now

**Review by raghunadhpokkalath — COMMENTED** [review]
Hi @ramiray  great job.LGTM
Can we please update readme also with ALB URL details

File: terraform/modules/networking/nacl-private.tf (hunk 1 above)
Comment (ramiray) [reply terraform/modules/networking/nacl-private.tf:32]: Hi Raghu , we have a separate rule for port  80 . 

resource "aws_network_acl_rule" "private_egress_http_vpc" {
  network_acl_id = aws_network_acl.private.id
  rule_number    = 300
  egress         = true
  protocol       = "tcp"
  rule_action    = "allow"
  cidr_block     = "10.0.0.0/16"
  from_port      = 80
  to_port        = 80
}

**Review by raghunadhpokkalath — DISMISSED** [review]

**Comment by ramiray** [comment]
Resolved conflict

**Review by raghunadhpokkalath — APPROVED** [review]

**Pull request merged**